src/summarycode/score.py

```diff
@@ -128,29 +128,107 @@ def compute_license_score(codebase):
     """
 
     scoring_elements = ScoringElements()
-    license_detections = get_field_values_from_codebase_resources(
-        codebase=codebase,
+    license_detections = get_field_values_from_resources(
+        codebase,
+        package_resources,
         field_name='license_detections',
         key_files_only=True,
     )
     license_match_mappings = get_matches_from_detection_mappings(license_detections)
     license_matches = LicenseMatchFromResult.from_dicts(license_match_mappings)
-    declared_license_expressions = get_field_values_from_codebase_resources(
-        codebase=codebase,
+    declared_license_expressions = get_field_values_from_resources(
+        codebase,
```

Comment:
We are still sending here both the codebase and the resources here, which is not required at all IMHO, send only the resources and this should be enough. Same for all other similar functions in `compute_license_score`

Reply:
I guess this is resolved now